portfolio/src/main/webapp/js/main.js

```diff
@@ -1,17 +1,45 @@
 // Copyright 2020 Tianyun Zhang
 
-// Fetch JSON array of comments and show then
 const loadComments = () => {
+  fetchComments();
+  fetchUserStatus();
+};
+
+// Fetch JSON array of comments and show them
+const fetchComments = () => {
   fetch('/comments')
     .then(res => res.json())
     .then(comments => {
       element = document.getElementById('comments');
       comments.forEach(comment => {
         node = document.createElement('p');
-        node.innerHTML = comment.name 
+        node.innerHTML = comment.name
+                          + '[' + comment.email + ']'
                           + ' @ ' + comment.date 
                           + ': ' + comment.content;
         element.appendChild(node);
       });
     });
 };
+
+// Fetch user status and show buttons.
+const fetchUserStatus = () => {
+  fetch('/user-status')
+    .then(res => res.json())
+    .then(status => {
+      console.log(status);
```

Comment:
Is this line used for debugging only? If yes, please remove it before submission.

Reply:
Oops I forgot to remove this debugging output. 😃 This line is removed now, please review again.